project/model.py

```diff
@@ -75,4 +78,4 @@ def shared_step(self, batch):
       return loss
 
     def configure_optimizers(self):
-      return SGD(self.parameters(), lr=2e-2)
+      return Adam(self.parameters(), lr=2e-2)
```

Comment:
Quick a high learning rate, use SGD with momentum 0.9, learning rate 0.03 and weight decay 1e-4